cluster/Makefile

```diff
@@ -1,8 +1,10 @@
 # Version and project can be overridden with the following:
 #  make target VERSION=0.XY PROJECT=foo-project
 
-VERSION?=0.13
+VERSION_TAG?=0.13
```

Comment:
Is this double defined with the common.sh definition below?

Reply:
Sort of? This sets an override-able default. I should probably just not set it here though.

Reply:
Removed